public/swagger/Build.json

```diff
@@ -64,6 +64,12 @@
     "updatedAt": {
       "type": "string"
     },
+    "url": {
```

Comment:
Conceptually `url` is the "internal" url and `viewLink` is the "external" url, I'm wondering if we don't want to serialize and send `url` anymore since we don't really want folks using it. Do we use it anywhere in the UI? Would need a corresponding change in the serializer.